helm-chart/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: nginx-service-mesh
 description: NGINX Service Mesh
-version: 0.7.0
```

Comment:
Any chance we could bump this to `2.0.0` to be aligned with the app version? The discrepancy is confusing.

Reply:
It is a little confusing, but it's the chart version, which is technically independent of the app version. See NIC as an example: https://github.com/nginxinc/kubernetes-ingress/blob/main/deployments/helm-chart/Chart.yaml

Reply:
Though I'm wondering if chart version should just be bumped to 0.8.0...I'll get back to that...

Reply:
I think we only bump major version of the chart if a `helm upgrade` cannot be performed. So until that happens, I'll actually just bump it to 0.8.0

Reply:
I looked up the best practices on chart versioning and found [this](https://codefresh.io/docs/docs/new-helm/helm-best-practices/#simple-1-1-versioning). Since our charts are managed alongside the app source code, I believe we should use the 1-1 versioning. There isn't a reason I can think of to manage a separate chart version in our case.

Reply:
But you are right about the charts being tied directly to the product.

Reply:
We could add a comment in the release notes explaining the change. A one-off confusing change is better than continuing a confusing versioning strategy IMO. Beside making our lives easier, it would also be a benefit to users when checking versions:
> This approach makes version bumping very easy (you bump everything up) and also allows you to quickly track what application version is deployed on your cluster (same as chart version).

Reply:
Makes sense. I made the change. I'll keep a note to include this in the release notes.